projects/demo/src/modules/components/island/examples/3/index.less

```diff
@@ -45,9 +45,10 @@
         border-radius-right-bottom: 0;
         border-bottom-right-radius: 0;
         border-bottom-left-radius: 0;
+    }
 
-        &:hover {
-            transform: translateY(-1.6rem);
-        }
+    &:hover [tuiaction] {
+        transition-delay: 250ms;
```

Comment:
This doesn't fix the issue, just delays it. It should either react to hover of the whole container, or have some :after element with height of 1.6rem to still be under the pointer.

Reply:
`react to hover of the whole container` -> & - is a container

Reply:
remove delay anyway